.github/actions/publish-versioned-docs/action.yml

```
@@ -0,0 +1,68 @@
+name: 'Publish Versioned Documentation'
+description: 'Publishes versioned documentation using Mike for MkDocs'
+inputs:
+  tag:
+    description: 'Version tag to publish (e.g., 1.0.0, 1.1.0)'
+    required: true
+  branch:
+    description: 'Branch to get documentation content from (defaults to current branch)'
+    required: false
+    default: ''
+  make_latest:
+    description: 'Make this version the latest/default'
+    required: false
+    default: 'true'
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.x
+        cache: pip
+
+    - name: Install dependencies
+      shell: bash
+      run: pip install -r requirements.txt
+
+    - name: Configure git for Mike
+      shell: bash
+      run: |
+        git config user.name "${{ github.actor }}"
+        git config user.email "${{ github.actor }}@scality.com"
```

Will show up more nicely in github, we'll get an actual binding to the user, otherwise we will get some randos

```suggestion
        git config user.email "${{ github.actor }}@users.noreply.github.com"
```

---

Changed in release.yaml